src/CookieAttributes.ts

```diff
@@ -1,36 +1,51 @@
-export interface CookieAttributes {
+export type CookieAttributes = BaseCookieAttributes & SameSiteCookieAttributes
+
+export interface BaseCookieAttributes {
```

Comment:
I don't think any of these interfaces other than `CookieAttributes` should be exported. They're an internal implementation detail.

Reply:
I thought maybe people would want to do type assertions, but you are right, that's unnecesary